vendor/bat-native-ads/src/bat/ads/internal/base/containers/container_util_unittest.cc

```diff
@@ -38,7 +38,7 @@ TEST(BatAdsContainerUtilTest,
   const std::vector<std::vector<std::string>> vectors = SplitVector(vector, 7);
 
   // Assert
-  const std::vector<std::vector<std::string>> expected_vectors = {vectors};
+  const std::vector<std::vector<std::string>>& expected_vectors = {vectors};
```

Comment:
& is not needed

Reply:
Local copy 'expected_vectors' of variable 'vectors' is never modified so we can avoid the copy

Reply:
442 lines without &, 292 lines with & using Compiler Explorer

Reply:
but this is a reference to a temporary object which doesn't needed in current context

Reply:
The test is wrong. Fixing the test now. `{vectors}` should have been `{vector}`